Задания с 25.02.2022/Task1/Task1/Trie.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+
+/// <summary>
+/// Структура "Бор"
+/// </summary>
+public class Trie
+{
+    private class Node
+    {
+        public Dictionary<char, Node> childs;
+        public bool isTerminal = false;
+
+        internal Node()
+        {
+            childs = new Dictionary<char, Node>();
+        }
+    }
+
+    private Node root = new Node();
+    public int Size = 0;
```

Comment:
Паблик-поля запрещены (можно только в private-вложенных классах, как Node, и то лучше свойства).
```suggestion
    public int Size { get; set; } = 0;
```